### ISSUE TITLE
Cannot find module error for scripts/install.js when installing.

### ISSUE THREAD

**Issue by smurrayatwork** [Prior99/native-image-diff#5]
When trying to install this module, I received an error related to node_modules/native-image-diff/scripts/install.js (stripped down paths just for an example):
```
> native-image-diff@0.1.4 install node_modules/native-image-diff
> node scripts/install.js

module.js:540
    throw err;
    ^

Error: Cannot find module 'node_modules/native-image-diff/scripts/install.js'
    at Function.Module._resolveFilename (module.js:538:15)
    at Function.Module._load (module.js:468:25)
    at Function.Module.runMain (module.js:684:10)
    at startup (bootstrap_node.js:187:16)
    at bootstrap_node.js:608:3
```

Switching back to version 0.1.3 fixes this.

**Comment by Prior99** [comment]
I will have a look ASAP. Sorry for the inconveniences.

**Comment by Prior99** [comment]
Should be fixed in 0.1.5

**Comment by aversini** [comment]
It did the trick, thanks @Prior99
